jupyter_fsspec/tests/test_api.py

```diff
@@ -539,3 +539,70 @@ async def test_upload_download(fs_manager_instance, jp_fetch):
     # downloaded_dirpath = local_root_path + '/some'
     # new_local_items = local_fs.ls(downloaded_dirpath)
     # assert downloaded_dirpath in new_local_items
+
+
+async def xtest_sync_push(fs_manager_instance, jp_fetch):
+    # WIP
+    fs_manager = fs_manager_instance
+    remote_fs_info = fs_manager.get_filesystem_by_protocol("s3")
+    remote_key = remote_fs_info["key"]
+    remote_fs = remote_fs_info["info"]["instance"]
```

Comment:
It's weird to me, that the call to get_filesystem succeeds without an exception but returns nothing.
Also, didn't we key these things by configured name rather than protocol?